personalize.api.php

```diff
@@ -353,26 +371,3 @@ function hook_personalize_get_executor_options() {
 function hook_personalize_option_load($option_set, $option) {
 
 }
-
-/**
- * Allows the alteration of any of the subforms within the personalize
- * campaign creation/editing flow.
- *
- * Options are:
- * - base
- * - variations (what)
- * - goals (why)
- * - targeting (why)
- * - scheduling (when)
- * - summary/reports (review)
- *
- * @param array $form
- *   The form that can be altered (passed by reference).
- * @param array $form_state
- *   The unalterable form state for context.
- * @param stdClass $agent_data
- *   The unalterable data for the current campaign for context.
- */
-function hook_personalize_campaign_wizard_OPTION_alter(&$form, $form_state, $agent_data) {
-
-}
```

Comment:
Well... not really... but yes.  It was forgotten when I pulled the functionality out cause we had two different ways of implementing things going on and weren't using this hook at all.